core/src/main/kotlin/io/fritz2/binding/store.kt

```diff
@@ -76,3 +113,28 @@ open class RootStore<T>(initialData: T, override val id: String = "", bufferSize
 
     override fun <X> sub(lens: Lens<T, X>) = SubStore(this, lens, this, lens)
 }
+
+
+interface LensId<T> {
```

Comment:
I am not sure, if the name explains, what it does. Maybe "ModelId"?

Reply:
Renamed in ModelId